pages/news/20XX-XX-XX-scons-cmake-migration.html

```diff
@@ -0,0 +1,27 @@
+title: SCons build system dropped in favor of CMake
+author: Holzhaus
+date: 2020-XX-XX 20:00:00
+tags: development, 2.3, beta, cmake, scons
+comments: yes
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown %}
+
+If you wanted to compile a recent version of Mixxx 2.3 beta or 2.4 yourself, you may have already noticed that we removed our old and rusty [SCons](https://scons.org/) setup in favor of [CMake](https://cmake.org/).
+
+[A few months ago]({% url "/news/2020-07-15-new-in-2-3-cmake.html" %}) we said that we were planning to ship Mixxx 2.3 with both SCons and CMake configurations (the latter would be considered "experimental").
+However, that turned out to be impractical.
+
+All of core developers started using CMake as primary build system since [initial support for it was added](https://github.com/mixxxdj/mixxx/pull/2280) more than a year ago.
```

Comment:
```suggestion
All of core developers started using CMake as their primary build system since [initial support for it was added](https://github.com/mixxxdj/mixxx/pull/2280) more than a year ago.
```